server/src/main/java/io/crate/types/IpType.java

```diff
@@ -35,6 +35,16 @@ public int id() {
         return ID;
     }
 
+    @Override
+    public String implicitCast(Object value) throws IllegalArgumentException, ClassCastException {
```

Comment:
could this replace `value` already,  or should we look later into stripping down `value`?

Reply:
I'd prefer to follow up on it with right after the `varchar(length)` implementation. Would it be ok?

Reply:
Sure :+1: